client/packages/cli/src/index.js

```diff
@@ -362,7 +362,7 @@ program
   .description('Push schema and perm files to production.')
   .action(async function (arg, inputOpts) {
     const ret = convertPushPullToCurrentFormat('push', arg, inputOpts);
-    if (!ret.ok) return;
+    if (!ret.ok) return process.exit(1);
```

Comment:
We don't technically need to write `return` with process.exit. 

Chose to specifically write it this way, because it's really jarring to see a function that that will implicitly stop control flow.